Assets/Scripts/Multiplayer/CoopCamera.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using UnityEngine;
+
+namespace Multiplayer
+{
+    public class CoopCamera : MonoBehaviour
+    {
+        public GameObject mainPlayer;
+        public GameObject coopPlayer;
+
+        public void Start()
+        {
+            if (Statics.gameMode == Statics.GameMode.SINGLEPLAYER)
```

Comment:
Couldn't you deactivate the Coop Camera when starting singleplayer?
For me, it currently looks like two cameras are active when playing alone, because
the transform of the coop cam is set to that of the main player.